docs/overview/script-storage.md

```diff
@@ -1,33 +1,52 @@
 ---
 title: Office Scripts file storage and ownership
 description: Information about how Office Scripts are stored in Microsoft OneDrive and transferred between owners.
-ms.date: 05/10/2022
+ms.date: 05/15/2022
 ms.localizationpriority: medium
 ---
 
 # Office Scripts file storage and ownership
 
-Office Scripts are stored as **.osts** files in your Microsoft OneDrive. They are stored separately from a workbook. To give others access, [share the script with an Excel workbook](excel.md#share-office-scripts). This means you're linking the script with the file, not attaching it. Whoever has access to the Excel file will also be able to view, run, or make a copy of the script.
+> [!IMPORTANT]
+> SharePoint support for Office Scripts is being rolled out and is not available to everyone. It's slowly released to larger numbers of users to ensure that it's working as expected. This feature is subject to change based on your feedback.
 
-Unless you share your scripts, no one else can access them. Your OneDrive settings control the shared access and permissions for all script **.osts** files, independent of any Excel settings. Scripts can't be linked from a local disk or custom cloud locations. Office Scripts only recognizes and runs a script if it's in your OneDrive folder or shared with the workbook.
+Office Scripts are stored as **.osts** files in your Microsoft OneDrive or a team SharePoint folder. They are stored separately from a workbook. To give others access, [share the script with an Excel workbook](excel.md#share-office-scripts). This means you're linking the script with the file, not attaching it. Whoever has access to the Excel file will also be able to view, run, or make a copy of the script.
```

Comment:
Should we say "to give others in the workbook access" since SharePoint gives others access by default?

Reply:
Good point. How about: "To give users who are outside of the SharePoint site access to the script" ?